packages/common/src/operations.graphql

```diff
@@ -178,3 +178,88 @@ query names(
     }
   }
 }
+
+query items(
+  $first: Int
+  $offset: Int
+  $key: ItemSortFieldInput!
+  $desc: Boolean
+) {
+  items(
+    page: { first: $first, offset: $offset }
+    sort: { key: $key, desc: $desc }
+  ) {
+    ... on ConnectorError {
+      __typename
+      error {
+        description
+        ... on DatabaseError {
+          __typename
+          description
+          fullError
+        }
+        ... on PaginationError {
+          __typename
+          description
+          rangeError {
+            description
+            field
+            max
+            min
+          }
+        }
+      }
+    }
+    ... on ItemConnector {
+      __typename
+      nodes {
+        __typename
+        availableBatches {
+          ... on ConnectorError {
+            __typename
+            error {
+              description
+              ... on DatabaseError {
+                __typename
+                description
+                fullError
+              }
+              ... on PaginationError {
+                __typename
+                description
+                rangeError {
+                  description
+                  field
+                  max
+                  min
+                }
+              }
+            }
+          }
+          ... on StockLineConnector {
+            nodes {
+              __typename
+              availableNumberOfPacks
+              batch
+              costPricePerPack
+              expiryDate
+              id
+              itemId
+              packSize
+              sellPricePerPack
+              storeId
+              totalNumberOfPacks
+              onHold
```

Comment:
I had to add `name` here, and to the `StockLineNode` in the schema: the Item query below as insisting that there should be a `name` on each available batch node